packages/utils/src/ts-eslint/Linter.ts

```diff
@@ -20,6 +20,7 @@ export type MinimalRuleModule<
 > = Partial<Omit<RuleModule<MessageIds, Options>, 'create'>> &
   Pick<RuleModule<MessageIds, Options>, 'create'>;
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Question :
Do we want to ignore declared things?
They're not technically values so using them in a typeof only isn't bad.

Reply:
I thought about it as well, but wasn't sure what to do

IMO it's ok not to report them ,WDYT?

Reply:
I think we should keep them reported by default, unless someone complains so we can reconsider. Can't think of a good use case for it at the moment other than global declarations.

Reply:
I'm ok with that 😄 

Maybe we can add an option in the future if needed